03-task-manager/starter/routes/tasks.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express')
+
+const router = express.Router()
+
+const {getAllTasks,createTask,getTask,updateTask,deleteTask} = require('../controllers/tasks')
+router.route('/').get(getAllTasks).post(createTask)
```

Comment:
I'm not a big fan of this approach of reusing the `route` calls for multiple endpoints because I think the advantage of reusing them isn't worth the loss in readability. Here's my suggestion on how to keep it simpler and easier to spot the operations:
```javascript
router.get("/", getAllTasks);
router.get("/:id", getTask);
router.post("/", createTask);
router.patch("/:id", updateTask);
router.delete("/:id", deleteTask);
```